Automattic/jetpack
# Site Editor: Change the `<` link to the theme showcase

## Proposed changes:

* Change the Site Editor's `<` link to the theme showcase page. See https://github.com/Automattic/wp-calypso/issues/80595.
* This PR works with https://github.com/WordPress/gutenberg/pull/54242 while this does not break anything without https://github.com/WordPress/gutenberg/pull/54242.

- [x] I'll create a follow-up PR to move this into `jetpack-mu-wpcom` https://github.com/Automattic/jetpack/pull/32913#discussion_r1318140030
	- -> https://github.com/Automattic/jetpack/pull/33024

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Apply https://github.com/WordPress/gutenberg/pull/54242 to your site (because this PR works with https://github.com/WordPress/gutenberg/pull/54242 while this does not break anything without it)
	* On Simple sites
		* Sync the change to your sandbox with AWE or rsync PCYsg-t5D-p2
	* On Atomic sites
		* Uninstall gutenberg
		* Checkout to https://github.com/WordPress/gutenberg/pull/54242
		* npm run dev
		* Create a new directory with including only `build/***`, `gutenberg.php`, `lib/***`, `packages/***`, `post-content.php`, `vendor/***`, and zip it
		* Upload the zip to `~/htdocs/wp-content/plugins` on your site 
		* Run `unzip gutenberg.zip` on your site and activate it
* Apply this patch to your site by doing https://github.com/Automattic/jetpack/pull/32913#issuecomment-1709419749
	* Note that you must set `define( 'JETPACK_MU_WPCOM_LOAD_VIA_BETA_PLUGIN', true );` to your wp-config.php on Atomic sites.
* Go to the Theme Showcase 
* Click the Live Preview option on the three-dots menu on the Theme Card
* Confirm if the `<` link point to the Theme Showcase page

<img width="341" alt="Screen Shot 2023-09-07 at 15 05 41" src="https://github.com/Automattic/jetpack/assets/5287479/ceca0a51-5f54-41cf-b1c9-87a379e3dc6a">

## Related

- https://github.com/Automattic/jetpack/pull/27601
- https://github.com/Automattic/wp-calypso/pull/81387
